python/tvm/relay/frontend/mxnet.py

```diff
@@ -1096,7 +1256,16 @@ def _from_mxnet_impl(symbol, shape_dict, dtype_info, mod=None):
                 dtype = dtype_info
             node_map[nid] = [_expr.var(node_name, shape=shape, dtype=dtype)]
         elif op_name in _convert_map:
-            res = _convert_map[op_name](children, attrs)
+            # Handle Control-Flow.
+            if op_name in ['_foreach', '_while_loop']:
+                subgraphs = node['subgraphs']
+                assert len(subgraphs) == 1, "only supports loop constructs with a single subgraph"
```

Comment:
This is not true for while_loop because it have 2 subgraphs, cond and body.

Reply:
Yeah I was trying to understand the code, I need to adapt for while loop. I got wrapped up working on paper for ASPLOS, resuming this tomorrow.